CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## CHANGELOG
 
+### 4.0.0
+> Update the routes. The basic route is now `/books-music-movies/recommendations/best-books/`. And `staff-picks`, `ya`, `childrens` are three lists on the same level of the structure.
+> Add the list selectors that help the user to choose the list by season/month or audience.
+> Add individual page for every single book for supporting legacy staff-picks links and homepage's staff-picks section. There won't be links to those pages from the main page.
```

Comment:
We converted the modal page into a static page, we didn't "add" any new feature for this.

Reply:
Hmm actually, I think `update` is a better word than `add` or `convert`.